packages/cli/src/test/compile.test.ts

```diff
@@ -172,13 +172,13 @@ msgstr ""
 
           const actualFiles = readFsToJson("/test")
 
+          mockFs.restore()
+
```

Comment:
Just found that one of the tests is always "One snapshot written", that's because mockFs wasn't restored before snapshotting.